### PR TITLE
[minor] Fixed GitHub unchanged files with check annotations

### PR DESCRIPTION
**Pull request by atb00ker**
** Still working on this, just tied up with other work! :smile:

### CODE REVIEW

**Comment by atb00ker** [comment]
** Still working on this, just tied up with other work! :smile:

**Comment by nemesisdesign** [comment]
> ** Still working on this, just tied up with other work!

Thank you Ajay! We'll try to help you with this when we can as well.

**Comment by atb00ker** [comment]
This is ready for review! :smile:

File: meta/main.yml [hunk 1]
```diff
@@ -12,22 +12,22 @@ galaxy_info:
   company: OpenWISP
   description: Official role to install and upgrade openwisp2 controller
   license: BSD
-  min_ansible_version: 2.5
+  min_ansible_version: 2.8
```
Comment (atb00ker) [inline meta/main.yml:15]: Versions below this one are not supported anymore.

**Review by nemesisdesign — APPROVED** [review]
Great! I guess there's nothing we can do for the remaining warnings, but at least now is a lot more readable.

**Review by pandafy — COMMENTED** [review]
Great work @atb00ker ! LGTM :+1: 

You should be able to silent `[line-length] line too long (174 > 160 characters)` by appending `# yamllint disable-line rule:line-length`.  

Please read [this section in the yamllint documentation](https://yamllint.readthedocs.io/en/stable/disable_with_comments.html#disabling-checks-for-a-specific-line) for more syntax options

Similarly, you can silent `[truthy] truthy value should be one of [false, true]` using `# yamllint disable-line rule:truthy`.

It can be merged after making these two changes. :smile:

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
> Great work @atb00ker ! LGTM 
> 
> You should be able to silent `[line-length] line too long (174 > 160 characters)` by appending `# yamllint disable-line rule:line-length`.
> 
> Please read [this section in the yamllint documentation](https://yamllint.readthedocs.io/en/stable/disable_with_comments.html#disabling-checks-for-a-specific-line) for more syntax options
> 
> Similarly, you can silent `[truthy] truthy value should be one of [false, true]` using `# yamllint disable-line rule:truthy`.
> 
> It can be merged after making these two changes.

Thanks for the suggestion @pandafy, @atb00ker, try that when you can so if you succeed we can get rid of all the warnings :+1: :+1:

**Comment by atb00ker** [comment]
Great this is done, I suppose! :smile:

**Review by pandafy — APPROVED** [review]
LGTM :+1: Thanks @atb00ker.

P.S: I realised that these errors/warning were generated by the [lint step of molecule](https://github.com/openwisp/ansible-openwisp2/runs/1522518589?check_suite_focus=true#step:6:22). GitHub was just putting them here alongside the diff.

**Review by nemesisdesign — APPROVED** [review]
Thank you very much @atb00ker :+1: :+1:

**Pull request merged**